com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/handler/LaunchRequestHandler.java

```diff
@@ -64,7 +77,8 @@
         }
 
         context.setAttached(false);
-        context.setSourcePaths(launchArguments.sourcePaths);
+        context.setVmStopOnEntry(launchArguments.stopOnEntry);
```

Comment:
Why do we care about the stopOnEntry flag? I thought we already disabled the logic somewhere else. Please try to focus on only one thing in a pull request.

Reply:
No, this code is always there, i just adjust the code position.